pygmt/tests/test_plot.py

```diff
@@ -302,12 +302,13 @@ def test_plot_sizes_colors_transparencies():
     return fig
 
 
-@pytest.mark.mpl_image_compare
+@pytest.mark.mpl_image_compare(filename="test_plot_matrix.png")
+@pytest.mark.parametrize("color", ["#aaaaaa", 170])
```

Comment:
Should we set `G="sequence"` so that people can input RGB or CMYK colors as a list like `[170, 170, 170]` or [0, 0, 0, 33] which gets converted to `170/170/170` and `0/0/0/33` respectively? Ref https://docs.generic-mapping-tools.org/6.2/gmtcolors.html#description.